Slim/Plugin/ExtendedBrowseModes/HTML/EN/plugins/ExtendedBrowseModes/settings/browsemodes.html

```diff
@@ -67,6 +67,25 @@
 		</table>
 	[% END %]
 
+	[% WRAPPER setting title="PLUGIN_EXTENDED_BROWSEMODES_TAGS" desc="PLUGIN_EXTENDED_BROWSEMODES_TAGS_DESC" %]
+		<table>
+			<tr>
+				<th>[% "PLUGIN_EXTENDED_BROWSEMODES_ROLE" | string %]</th>
+				<th>[% "PLUGIN_EXTENDED_BROWSEMODES_ROLE_TEXT" | string %]</th>
+			</tr>
+			[% FOREACH tag = customTags.keys.sort %]
+				<tr>
+					<td><input type="text" name="[% tag %]_tag" value="[% tag | html %]" class="stdedit"/></td>
```

Comment:
Please send `tag` through the `html` filter for the name, too.

Reply:
Not sure how to do this, nesting like this doesn't work:

`name="[% [% tag %]_name | html %]"`

Reply:
I'm about to be off for a while... I think what you want to do is  `name="[% tag | html %]_tag" value...`?

Reply:
Ah yes, easy when you know how!

Push incoming.